docker-compose.yaml

```diff
@@ -0,0 +1,61 @@
+version: "3.6"
+services:
+  postgres:
+    image: postgres:11
+    container_name: postgres
+    ports:
+      - 5432:5432
+    restart: always
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: postgrespassword
+  graphql-engine:
+    image: hasura/graphql-engine:v1.2.1.cli-migrations-v2
+    container_name: graphql-engine
+    ports:
+      - 8080:8080
+    depends_on:
+      - postgres
+    restart: always
+    volumes:
+      - ./graphql-server/migrations:/hasura-migrations
+      - ./graphql-server/metadata:/hasura-metadata
+    environment:
+      HASURA_GRAPHQL_DATABASE_URL: postgres://postgres:postgrespassword@postgres:5432/postgres
+      HASURA_GRAPHQL_ENABLE_CONSOLE: "true" # set to "false" to disable console
+      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
+      HASURA_GRAPHQL_ADMIN_SECRET: myadminsecretkey
+      HASURA_GRAPHQL_UNAUTHORIZED_ROLE: anonymous
+      ACTION_BASE_ENDPOINT: http://actions-server:3001
```

Comment:
I tried a few different things here (i.e. `localhost`, `*:3000`, etc), but this isn't working at the moment:

```
Unhandled Rejection (Error): GraphQL error: http exception when calling webhook
```

<details>
<summary><code>graphql-engine</code> error</summary>

```json
{
  "type": "http-log",
  "timestamp": "2020-05-22T23:35:50.273+0000",
  "level": "error",
  "detail": {
    "operation": {
      "user_vars": {
        "x-hasura-role": "anonymous"
      },
      "error": {
        "internal": {
          "error": {
            "type": "http_exception",
            "message": "ResponseTimeout"
          },
          "response": null,
          "request": {
            "body": {
              "session_variables": {
                "x-hasura-role": "anonymous"
              },
              "input": {
                "gameId": "1b342dd1-cf34-4d6c-8c0a-c32134b103b3",
                "clientUuid": "9507226f-3fd8-4578-bae2-d6857abaf527"
              },
              "action": {
                "name": "joinGame"
              }
            },
            "url": "http://actions-server:3000/joinGame",
            "headers": []
          }
        },
        "path": "$",
        "error": "http exception when calling webhook",
        "code": "unexpected"
      },
      "request_id": "daa4e0e1-f3aa-494e-984e-e9a430c8a83d",
      "response_size": 436,
      "query": {
        "variables": {
          "gameId": "1b342dd1-cf34-4d6c-8c0a-c32134b103b3",
          "clientUuid": "9507226f-3fd8-4578-bae2-d6857abaf527"
        },
        "operationName": "JoinGame",
        "query": "mutation JoinGame($gameId: String!, $clientUuid: String!) {\n  joinGame(gameId: $gameId, clientUuid: $clientUuid) {\n    id\n    jwt_token\n    __typename\n  }\n}\n"
      }
    },
    "http_info": {
      "status": 200,
      "http_version": "HTTP/1.1",
      "url": "/v1/graphql",
      "ip": "172.21.0.1",
      "method": "POST",
      "content_encoding": null
    }
  }
}
```
</details>